docs/developers/protocol/hermez-protocol/circuits/circuits.md

```diff
@@ -457,7 +462,7 @@ It should be noted that in L1 tx, no errors are allowed but the circuit needs to
 |        amount        | uint192 |              amount to transfer from L2 to L2               |
 |      loadAmount      | uint192 |               amount to deposit from L1 to L2               |
 |     feeSelector      |  uint8  |                      user selector fee                      |
-|       onChain        |  bool   |          determines if the transaction is L1 or L2           |
+|       onChain        |  bool   |          determines if the transacion is L1 or L2           |
```

Comment:
(this one was already correct previous to this PR)
```suggestion
|       onChain        |  bool   |          determines if the transaction is L1 or L2           |
```

Reply:
Update has been made with an old version table.
Fixed.